pkgs/by-name/na/navidrome/package.nix

```diff
@@ -16,18 +16,18 @@
   ffmpegSupport ? true,
 }:
 
-buildGo123Module rec {
+buildGo124Module rec {
```

Comment:
Is there a reason Go needs to be pinned to 1.24? In the documentation I only found a requirement on Go 1.22+, so using `buildGoModule` could suffice.

Reply:
I've issued a PR to navidrome's docs to update the go requirement https://github.com/navidrome/website/pull/199